iFixit/checkout-ui-extensions
# Clear Out Cart On Ifixit API

## Overview

For headless stores that is, we now send a call to the ifixit api when the order has been completed to delete the users cart.

## QA

Make sure that user's cart is cleared, make sure this only fires on the thank you page.

Closes: https://github.com/iFixit/checkout-ui-extensions/issues/36
